app/uk/gov/hmrc/gform/submission/DestinationAuditRepo.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.submission
+
+import reactivemongo.api.DefaultDB
+import uk.gov.hmrc.gform.repo.Repo
+
+class DestinationAuditRepo(mongo: () => DefaultDB)
+    extends Repo[DestinationAudit]("destinationAudit", mongo, _.formId.value)
```

Comment:
Who and when is writing into this collection? How will this affect MDTP version?

Reply:
Yes. I don't believe this will be used when the service is working on MDTP. This will be used in the Ofsted environment. During an Ofsted forms lifecycle certain events will be written to this collection.